boards/xtensa/intel_adsp_cavs18/doc/generic_intel_adsp.rst

```diff
@@ -61,6 +61,17 @@ also specify the rimage tool, config and key by:
               --config-dir [path to dir of .toml config file] \
               --key [path to signing key]
 
+
+Note: Sometimes you might have multiple network interfaces. In this case,
+if the cavstool.py cannot start with a correct IP for you, you can check
+your IP address by "ip a" or "ifconfig" command, then specify correct the
+IP address by:
+
+.. code-block:: console
+
+   sudo ./cavstool.py -s 192.168.0.2
+
+
```

Comment:
In the future, the script will be fixed and will listen on all interfaces by default and this option will allow you to bind to a specific interface if you need to.

Reply:
very good suggestion, working on it.

Reply:
Added specifying a log port and req port for the cavs server:
```
cavstool.py -s {host}:{log port} --req-port {req port} 
OR 
cavstool.py -s {host} --log-port {log port} --req-port {req port}
```